packages/react-reconciler/src/ReactDebugFiberPerf.js

```diff
@@ -119,11 +120,19 @@ const getFiberLabel = (
   }
 };
 
+const getFiberName = (fiber: Fiber): string | null => {
+  if (fiber.tag === SimpleMemoComponent) {
+    return getComponentName(fiber.elementType);
```

Comment:
Tested

```diff
-getComponentName(someFiber.type)
+getComponentName(someFiber.elementType)
```
on all source files in `master` and no test broke. This may be the better option if other types of Fibers change their `type` during work i.e. `getComponentName` should only be called with element types not fiber types (enforceable with types?).

Reply:
#15638 would remove the necessity for this branch.